pkg/kubelet/cm/pod_container_manager_linux.go

```diff
@@ -63,7 +63,11 @@ var _ PodContainerManager = &podContainerManagerImpl{}
 // Exists checks if the pod's cgroup already exists
 func (m *podContainerManagerImpl) Exists(pod *v1.Pod) bool {
 	podContainerName, _ := m.GetPodContainerName(pod)
-	return m.cgroupManager.Exists(podContainerName)
+	if err := m.cgroupManager.Validate(podContainerName); err != nil {
+		klog.V(4).InfoS("Validate cgroup error", "podContainerName", podContainerName, "err", err)
```

Comment:
It's not very clear from the log message that podContainerName is a cgroup name. Can you refrase it somehow?

Reply:
Done. Thanks for review comments.